### DISCUSSION TITLE
Removing bot from player count

### DISCUSSION THREAD

**Discussion by cgirerd** [tltneon/lgsl#50] [Q&A] [answered]
Hello!
Thanks a lot for the nice work! I have one question: I am using this code for a Counter Strike: Condition Zero server (it runs with the Half-Life Steam Protocol), and the bots happen to be counted in the total number of players. Do you know if there would be a way to remove the bots from the player count?
Best,
Cedric

**Comment by tltneon** [comment, marked as answer]
Hello, @cgirerd !
There is a solution:
Add `$server['s']['players'] -= $server['e']['bots'];` after line 1179 into **lgsl_files/lgsl_protocol.php**

![](https://imgur.com/JQnoK1h.png)

**Reply by cgirerd** [reply]
Thanks a lot for your very fast reply! that works very well :)
